RecoMuon/Configuration/python/DisplacedMuonSeededStep_cff.py

```diff
@@ -59,7 +60,7 @@
 #for displaced global muons
 muonSeededTracksOutInDisplacedClassifier = RecoTracker.IterativeTracking.MuonSeededStep_cff.muonSeededTracksOutInClassifier.clone()
 muonSeededTracksOutInDisplacedClassifier.src='muonSeededTracksOutInDisplaced'
-
+eras.trackingPhase2PU140.toModify( muonSeededTracksOutInDisplacedClassifier , vertices = "pixelVertices")
```

Comment:
Do this change here instead
https://github.com/cms-sw/cmssw/blob/CMSSW_8_1_X/RecoTracker/FinalTrackSelectors/python/TrackCutClassifier_cff.py
As `TrackCutClassifier` gets cloned in various places, this way reduces copy-paste.